src/model/parsers/__tests__/index-test.ts

```diff
@@ -398,6 +398,48 @@ describe('Model Parsers', () => {
     });
   });
 
+  describe('Unique Parser', () => {
+    it('should parse one unique node with one child tag', () => {
+      const el = xmlUtil.toElement(`
+        <mock ${yinNS}>
+          <yin:unique tag="foo"/>
+        </mock>
+      `);
+
+      expect(Array.from(Parsers.UniqueParser.parse(el).values())).to.deep.equal(['foo']);
+    });
+
+    it('should parse multiple unique nodes with one child tag', () => {
+      const el = xmlUtil.toElement(`
+        <mock ${yinNS}>
+          <yin:unique tag="foo"/>
+          <yin:unique tag="bar"/>
+        </mock>
+      `);
+
+      expect(Array.from(Parsers.UniqueParser.parse(el).values())).to.deep.equal(['foo', 'bar']);
+    });
+
+    it('should parse multiple unique nodes with multiple child tags', () => {
+      const el = xmlUtil.toElement(`
+        <mock ${yinNS}>
+          <yin:unique tag="foo bizz"/>
+          <yin:unique tag="bar"/>
+        </mock>
+      `);
+
+      expect(Array.from(Parsers.UniqueParser.parse(el).values())).to.deep.equal(['foo', 'bizz', 'bar']);
```

Comment:
huh, this is an interesting case...
`foo` and `bizz` by themselves are not required to be unique, just the pair of them together. So if you got just the `foo` node and called `isUnique()` on it, it'd return true, but that's not strictly correct.

I guess for our planned usage that's okay, but something to keep in mind